backend/src/tasks/dnstwist.ts

```diff
@@ -30,13 +28,45 @@ async function runDNSTwist(domain: Domain) {
 export const handler = async (commandOptions: CommandOptions) => {
   const { organizationId, organizationName } = commandOptions;
 
+  await connectToDatabase;
+  const date = new Date(Date.now());
   console.log('Running dnstwist on organization', organizationName);
   const domainsWithIPs = await getIps(organizationId);
 
   const vulns: Vulnerability[] = [];
   for (const domain of domainsWithIPs) {
     try {
       const results = await runDNSTwist(domain);
+      const existingVulns = await Vulnerability.find({
+        domain: { id: domain.id },
+        source: 'dnstwist'
+      });
+      let existingDomains: object[] = [];
+      const newDomainNames: String[] = [];
+      if (existingVulns.length > 0) {
+        existingDomains = existingVulns[0].structuredData['domains'];
+      }
+      for (const newDomain of results) {
+        newDomain['date-observed'] = date;
+        newDomainNames.push(newDomain['domain-name']);
+
+        if (!existingDomains) {
+          existingDomains = [newDomain];
+          // else if dnstwist domain has already been added
+        } else if (
+          existingDomains.some(
+            (oldDomain) => oldDomain['domain-name'] === newDomain['domain-name']
+          )
+        ) {
+          continue;
+        } else {
+          existingDomains.push(newDomain);
+        }
+      }
+      //filter out any domains that no longer exist
```

Comment:
For consistency, can you make sure the comments start with "// " (with a space) and are capitalized?
```suggestion
      // Filter out any domains that no longer exist
```